src/series/Castlevania.yml

```diff
@@ -86,27 +86,53 @@ randomizers:
     updated-date: 1900-01-01
     added-date: 1900-01-01
 -   games:
-    - 'Castlevania: Symphony of the Night (PlayStation / Saturn)'
-    identifier: 3snowp7im's Area Randomizer
+    - 'Castlevania: Symphony of the Night (PlayStation)'
+    identifier: Castlevania: SOTN Randomizer (Launcher Application by SacredLucy)
```

Comment:
I believe you need to wrap this line in quotes

Reply:
yea I just fixed that, as well as some issues with game titles